src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -2462,6 +2462,7 @@ bool Eltwise::canFuseConvert(const NodePtr& convertNode) {
                                              {},
                                              {convertNode->getOriginalOutputPrecisionAtPort(0)});
 #else
+    (void)this;
```

Comment:
It seems we can just remove the whole #ifdef logic

Reply:
Done